pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[tool.black]
```

Comment:
`pyproject.toml` seems to be where the Python community is moving for configure tools such as `black`. As far as I know, it's the only sensible way to configure Black (unless we specify everything on the command line - which is pretty verbose). Since it exists for that reason, putting other tool config in it where possible seemed sensible, too.

Reply:
Sounds good.